### PR TITLE
Wrapper: include in ChartWrapper component

### LINKED ISSUE #100: wrap the chart with the propper css
Latest release somehow impacted the way the charts are mounted I believe

before v0.3.2, I was able to render the charts in overlord, with react-grid-layout wrapper, without any special CSS on their `div`s. Now, I always need to have the [charts wrapped in a div that has (mandatory) `display: flex`](https://github.com/EQWorks/overlord/blob/5b86f775159eab47ebe6353d5f02e39830d41a71/src/campaign/report/tab/tab.js#L138-L157) otherwise chart won't render, not even if I add `height` and `width` in the `div` style

I've tried to return the config and files to what it was before this merge https://github.com/EQWorks/chart-system/pull/96 as I thought it had introduced this behaviour, but even by putting it back, I experienced the same thing: the need to have `display:flex` for the chart to render.

any idea why or, more important, is there a way for us to include it in the chart itself so there's no need to add this css on library usage?

context starts [here](https://eqworks.slack.com/archives/GV827LS0M/p1597854836048000?thread_ts=1597848484.043600&cid=GV827LS0M)
 but more specifically [here](https://eqworks.slack.com/archives/GV827LS0M/p1597859215059000?thread_ts=1597848484.043600&cid=GV827LS0M
)

### PR DESCRIPTION
**Pull request by geoerika**
Closes #100
- chart-wrapper/index.js: included Wrapper styled component
- responsive-chart-wrapper.js: replaced Wrapper with simpler ChartDiv styled component
- CHANGELOG.md: added Wrapper changes and revised

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,19 +9,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Legend - label trimming when special characters are present in the labels
 
+### Changed
+- `<ChartWrapper \>` - moved all necessary chart wrapper css in ChartWrapper component
```
Comment (woozyking) [inline CHANGELOG.md:13]: the actual [self-closing tag](https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/self-closing-comp.md) should be `<ChartWrapper />`

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,19 +9,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
 ## [0.3.2] - 2020-08-18
 ### Fixed
-- ReadMe - update scatter & line props usage
-- Utils - update sort function props to be more flexible
+- ReadMe - updated scatter & line props usage
+- Utils - updated sort function props to be more flexible
```
Comment (woozyking) [inline CHANGELOG.md:18]: no need to introduce unnecessary changes like these, when both present and past tenses are okay.

also, writing git messages in present tense is considered a [normal convention](https://stackoverflow.com/a/3580764/158111) (as `git` itself promotes). and combined with the the past tense headers like `Fixed`, `Changed` in the CHANGELOG itself, it's perfectly normal to write it as:

```
Did:
- utils - update ...
```

for "did update" makes more sense than, "did updated".

Comment (geoerika) [reply CHANGELOG.md:18]: LOL I looked over react-labs CHANGELOG and wanted to be consistent as we used both. You need me to revert?

File: CHANGELOG.md (hunk 1 above)
Comment (tlowande) [reply CHANGELOG.md:13]: I feel like this should go under unreleased for now right?

**Pull request merged**